weave/tests/test_weave_client.py

```diff
@@ -288,7 +288,7 @@ async def score(target, model_output):
         return target == model_output
 
     evaluation = Evaluation(
-        name="my-eval",
+        metadata={"name": "my-eval"},
```

Comment:
this feels very odd to me - does it feel odd to you?

Reply:
Why?

Reply:
well, as a user, i am thinking to myself: i want to name this evaluation, why is it some nested dictionary? It feels like the underlying data model is leaking into user-land. If we didn't have this name collision issue, we would not choose this API. There might not be a good alt, just want to discuss / see if there are other ideas

Reply:
It feels right to me.  This dict format is just taking advantage of pydantic so you don't need to use the class.  You could alternatively write:
```python
metadata=Metadata(name="my-eval")
```

That said, I wonder if having a very general metadata dict is also useful -- basically simple k,v tags for objects

Reply:
For `Evaluation` specifically, we could also have `name` as the top-level then forward the value to `metadata`